src/msw/window.cpp

```diff
@@ -3598,7 +3599,63 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
         case WM_ENDSESSION:
             processed = HandleEndSession(wParam != 0, lParam);
             break;
+        case WM_ENTERIDLE:
```

Comment:
I have no idea if doing this here is where it should be done.

But I wonder if we should do this:
1. For every idle message?
2. When not in dark mode?
3. If not Windows 11?

Reply:
I agree with the points (2) and (3), i.e. we need to restrict this to dark mode under Windows 11 (and later). Point (1) is not relevant, I think, as we don't do this on every idle message, but only when entering a local message loop.

OTOH I'd like to add another point: it would be nice to move this into a separate function.